book/docs/.ipynb_checkpoints/Open_data_code-checkpoint.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p><code>12 Use version control</code> needs to be higher in the list  since several other items rely on this. <code>gitignore</code> isn't mentioned anywhere, leaving out things like large data, ipynb_checkpoints, secrets, etc... </p><p> </p><p>I almost wonder if this list is overwhelming as written (tbd later), things like Flake8 are often woven into pre-commit or CI pipelines.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Moved version control a bit further up, added notes on use of `.gitignore` . I think we can decide on things like Flake8 later.